scale-tests/deploy-dispatchers.py

```diff
@@ -0,0 +1,58 @@
+import shakedown
+import sys
+
+
+# This script will deploy the specified number of dispatchers with an optional
+# options json file. It will take the given base service name and
+# append an index to generate a unique service name for each dispatcher.
+#
+# The service names of the deployed dispatchers will be written into an output
+# file.
+#
+# Running:
+# > dcos cluster setup <cluster url>
+# > python deploy-dispatchers.py 2 spark-instance dispatchers.out
+
+
+SPARK_PACKAGE_NAME="spark"
+
+
+#TODO: add options json file contents
+def deploy_dispatchers(num_dispatchers, service_name_base, output_file, options_json_file):
+    outfile = open(output_file, "w")
+
+    with open(output_file, "w") as outfile:
+        for i in range(0, num_dispatchers):
+            service_name = "{}-{}".format(service_name_base, str(i))
+            options = {}
+            options["service"] = options.get("service", {})
+            options["service"]["name"] = service_name
+
+            options["service"]["UCR_containerizer"] = True
+            options["service"]["docker-image"] = "mesosphere/spark-dev:931ca56273af913d103718376e2fbc04be7cbde0"
+
+            shakedown.install_package(
+                SPARK_PACKAGE_NAME,
+                options_json=options)
```

Comment:
You take in `options_json_file` as an arg but don't seem to use it. Should it be used here / default to your `options` if not present?

Reply:
True, the arg is currently unused. @sschneid will fix this in SPARK-724.